src/couch_file.erl

```diff
@@ -216,24 +216,42 @@ close(Fd) ->
     gen_server:call(Fd, close, infinity).
 
 
-delete(RootDir, Filepath) ->
-    delete(RootDir, Filepath, true).
-
-
-delete(RootDir, Filepath, Async) ->
+delete(RootDir, Filepath, _Async, true) ->
```

Comment:
@iilyak I think we'd better just to make third attribute here into options' list and pass both async and rename_on_delete flags in it, instead of growing arity. or am I missing something?